src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -305,6 +307,36 @@ protected void setInteractionState(DcMsg messageRecord, boolean pulseHighlight)
     }
   }
 
+  private void setContentDescription(DcMsg messageRecord) {
+    String desc = "";
+    if (groupSenderHolder.getVisibility() == View.VISIBLE) {
+      desc = groupSender.getText() + "\n";
+    }
```

Comment:
add an empty line to mark end of logic statements and to make sure no one thinks an `else` is missing :)

```suggestion
    }

```